src/SearchResults.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+function SearchResults(props) {
+  return (
+    <div>
+      <table className="table table-bordered">
+        <tr>
+          <th scope="col">Id</th>
+          <th scope="col">Title</th>
+          <th scope="col">First Name</th>
+          <th scope="col">Surname</th>
+          <th scope="col">Email</th>
+          <th scope="col">Room Id</th>
+          <th scope="col">Check in Date</th>
+          <th scope="col">Check out Date</th>
+        </tr>
```

Comment:
Don't forget about the 7th exercise. You should add a column that shows the number of nights for each booking.
https://github.com/CodeYourFuture/cyf-hotel-react#7-calculate-and-show-the-number-of-nights-for-each-booking